packages/block-library/src/navigation/block.json

```diff
@@ -9,7 +9,10 @@
 	"textdomain": "default",
 	"attributes": {
 		"navigationMenuId": {
-			"type": "number"
+		  "type": "number"
+		},
+		"menuSlug": {
```

Comment:
Why not just "slug" since we're on the navigation block already.

Reply:
good point

Reply:
also just "slug" is what template parts do